### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by ChristineStawitz-NOAA** [NOAA-FIMS/FIMS#534] [Announcements]
## What's Changed

Summary of high level/breaking changes:
* Added a `pkgdown` site for FIMS
* Add code cov tests and site link
* Remove recruitment bias correction and adjustment because we plan to use `TMB` (https://github.com/NOAA-FIMS/FIMS/issues/185)
* Improve logging coverage and separate logs into multiple files
* Put recruitment deviations on the log scale when input
* Report more values with standard errors
* Code cleanup, documentation, and consistency improvements
* Remove rapidjson dependence
* Add `start_year` and `end_year` as slots to FIMSFrame
* Make `log_obs_error` a vector
* Load the Rcpp module upon package install so components don't need to be prefaced by `fims$`
* Refactor vectors to use `fims::Vector` class
* Add Newton steps to FIMS integration test to improve convergence in integration test

More detail:
* Update issue forms by @k-doering-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/426
* Update DESCRIPTION to match release number by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/425
* resolved issues causing warnings by @MOshima-PIFSC in https://github.com/NOAA-FIMS/FIMS/pull/430
* #307:  export age comp function by @k-doering-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/438
* Style code and document by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/439
* 368 convert fims to a pkgdown site by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/437
* #242 fix codecov calc and upload by @k-doering-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/436
* 391 feature find a file parser for c++ tests by @msupernaw in https://github.com/NOAA-FIMS/FIMS/pull/434
* Style code and document by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/440
* fix: deploying pkgdown by @k-doering-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/448
* fix: remove set() by @KyleShertzer-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/451
* Update DESCRIPTION by @peterkuriyama-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/435
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/454
* Remove bias correction/adjustment by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/452
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/460
* Style code and document by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/458
* doc: error and progress logging for information.hpp by @nathanvaughan-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/424
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/461
* fix: refactor parameter constructor by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/455
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/462
* doc: logit_steep correction by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/463
* refactor add_tmb_to_fims_internal by @msupernaw in https://github.com/NOAA-FIMS/FIMS/pull/459
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/465
* Created get-gtest-codecov to create coverage.xml report for issue #242 by @MOshima-PIFSC in https://github.com/NOAA-FIMS/FIMS/pull/470
* #441: fix reviewer checklist build trigger, edit checklist by @k-doering-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/474
* 377 refactor call report f from modelhpp instead of fleet and population by @Andrea-Havron-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/466
* Style code and document by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/484
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/483
* Changed <T> to <Type> by @MOshima-PIFSC in https://github.com/NOAA-FIMS/FIMS/pull/487
* 411 refactor naming convention of member and local variables in the fleet class by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/476
* Remove test data from fims deterministic folder by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/481
* Update GHA workflows to enforce FIMS installation by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/489
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/488
* renamed dimension folded indices to be clearer by @peterkuriyama-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/480
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/490
* rm acomp/index nll tests objects by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/491
* Delete test files by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/492
* fix: update windows-2019 to windows-latest by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/495
* Create multiple namespaces in FIMS by @k-doering-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/479
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/500
* fix: use expect_equal() for deterministic tests by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/496
* Style code and document by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/505
* fix: build trigger only on pr being opened by @k-doering-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/498
* 407 updating log files by @nathanvaughan-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/499
* Name refactoring for tests, growth, and median by @nathanvaughan-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/503
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/509
* Fix the test for population.CalculateRecruitment() by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/508
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/512
* fix: update run google test to use windows latest by @k-doering-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/511
* refactor: move duplicated code into the test fixture section by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/497
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/513
* Fims concurrent by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/520
* Remove third party library rapidjson by @Bai-Li-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/519
* Make Doxygen comment style the same by @k-doering-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/518
* refactor recruitment deviations to log space by @Andrea-Havron-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/514
* add documentation to test-fims-estimation by @Andrea-Havron-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/515
* Added start_year and end_year as slots to FIMSFrame by @kellijohnson-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/501
* New 464 load module by @Andrea-Havron-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/526
* make log_obs_error a vector by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/532
* Fims vector by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/517
* format C++ code with clang format by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/523
* Style code and document by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/522
* Release mq by @ChristineStawitz-NOAA in https://github.com/NOAA-FIMS/FIMS/pull/533

## New Contributors
* @peterkuriyama-NOAA made their first contribution in https://github.com/NOAA-FIMS/FIMS/pull/435

**Full Changelog**: https://github.com/NOAA-FIMS/FIMS/compare/v0.1.0...v0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/NOAA-FIMS/FIMS/releases/tag/v0.2.0'>v0.2.0</a>.</em>

**Comment by kellijohnson-NOAA** [comment]
I am closing this announcement as we are on version 0.9.3 and we haven't been using the Discussion board as a place to repost release notes. We will be thinking about how to announce releases in the immediate future. Stay tuned 😄.
